irods/test/access_test.py

```diff
@@ -444,7 +444,12 @@ def test_iRODSAccess_cannot_be_constructed_using_unsupported_type__issue_558(sel
         # Before the fix in #558, this would have been allowed and only later would the type discrepancy be revealed,
         # leading to opaque error messages. Now, the types are checked on the way in to ensure clarity and correctness.
         # TODO(#480): We cannot use the unittest.assertRaises context manager as this was introduced in python 3.1.
```

Comment:
Does this comment need to be adjusted to fit the change?

Reply:
I personally think it's fine. We still prefer to use `unittest.assertRaises` as a context manager over either of the current options and will replace this with the context manager in the migration to python3-only.

Reply:
Agree. resolving.